Tangem/Modules/Send/SendTransactionDispatcher/CommonSendTransactionDispatcher.swift

```diff
@@ -26,7 +26,7 @@ class CommonSendTransactionDispatcher {
 // MARK: - SendTransactionDispatcher
 
 extension CommonSendTransactionDispatcher: SendTransactionDispatcher {
-    func send(transaction: SendTransactionType) async throws -> SendTransactionDispatcherResult {
+    func send(transaction: SendTransactionType, updateWalletAfterSending: Bool) async throws -> SendTransactionDispatcherResult {
```

Comment:
я бы вот так предложил лучше
```suggestion
    func send(transaction: SendTransactionType, options: SendTransactionDispatcherOptions?) async throws -> SendTransactionDispatcherResult {
```

```
enum SendTransactionDispatcherOptions {
    case updateWallet
}

Reply:
поправил